doc/api/globals.md

```diff
@@ -17,6 +17,83 @@ The objects listed here are specific to Node.js. There are [built-in objects][]
 that are part of the JavaScript language itself, which are also globally
 accessible.
 
+## Class: `AbortController`
+<!--YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+<!-- type=global -->
+
+A utility class used to signal cancelation in selected `Promise`-based APIs.
+The API is based on the Web API [`AbortController`][].
+
+```js
+const ac = new AbortController();
+
+ac.signal.on('abort', () => console.log('Aborted!'));
+
+ac.abort();
+
+console.log(ac.signal.aborted);  // Prints True
+```
+
+### `abortController.abort()`
+<!-- YAML
+added: REPLACEME
+-->
+
+Triggers the abort signal, causing the `abortController.signal` to emit
+the `'abort'` event.
+
+### `abortController.signal`
+<!-- YAML
+added: REPLACEME
+-->
+
+* Type: {AbortSignal}
+
+### Class: `AbortSignal extends NodeEventEmitter`
+<!-- YAML
+added: REPLACEME
+-->
+
+#### Event: `'abort'`
+<!-- YAML
+added: REPLACEME
+-->
+
+The `'abort'` event is emitted when the `abortController.abort()` method
+is called. The callback is invoked with a single object argument with a
+single `type` propety set to `'abort'`:
+
+```js
+const ac = new AbortController();
+
+// Use either the onabort property...
+ac.signal.onabort = () => console.log('aborted!');
```

Comment:
Would this override another listener on the signal? If yes, that may not be the best to recommend in the docs?

Reply:
I believe it would only override a listener previously set with `onabort`, not with `addEventListener()`.

Reply:
Then I think using `onabort` should be discouraged in general (and not shown in the docs). Another reason to discourage it is that `{ once: true }` should always be passed or you may get memory leaks (and `abort` can never be fired more than once).

Reply:
It only overrides any previously set `onabort`. All other abort listeners are untouched.